docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -1,12 +1,13 @@
 = Cloud Native Applications
-:github: https://github.com/spring-cloud/spring-cloud-commons
+:githubroot: https://github.com/spring-cloud
+:github: {githubroot}/spring-cloud-commons
 :githubmaster: {github}/tree/master
 :docslink: {githubmaster}/docs/src/main/asciidoc
 :toc:
 
 include::intro.adoc[]
 
-include::https://raw.githubusercontent.com/spring-cloud/spring-cloud-build/master/docs/src/main/asciidoc/contributing-docs.adoc[]
+include::https://github.com/spring-cloud/spring-cloud-build/blob/master/docs/src/main/asciidoc/contributing-docs.adoc[]
```

Comment:
As far as I can see this is the only place we try to include content via a url in our main docs. The readme's do it, but they get processed differently. Maybe this should just be removed or copied in for now until that get's figured out.